app/controllers/spree/admin/bolts_controller.rb

```diff
@@ -30,7 +30,7 @@ def bolt_configuration_params
           .require(:solidus_bolt_bolt_configuration)
           .permit(
             :bearer_token,
-            :environment_url,
+            :test,
```

Comment:
The `test` name sounds a little bit generic to me and can be misleading. WDYT about using a more specific name?

Reply:
what do you think of `sandbox`?

Reply:
Yeah I think it's better